fbw-a32nx/src/systems/fmgc/src/navigation/Navigation.ts

```diff
@@ -198,14 +198,34 @@ export class Navigation implements NavigationProvider {
     NearbyFacilities.getInstance().setPpos(this.ppos);
   }
 
+  public updateRnp(rnp: number | null) {
+    if (rnp) {
+      this.requiredPerformance.setPilotRnp(rnp);
+    } else {
+      this.requiredPerformance.clearPilotRnp();
+    }
+  }
+
+  public isRnpManual(): boolean {
+    return this.requiredPerformance.manualRnp;
+  }
+
+  public isAcurracyHigh(): boolean {
+    return this.accuracyHigh;
+  }
+
   public getBaroCorrectedAltitude(): number | null {
     return this.baroAltitude;
   }
 
-  public getEpe(): number {
+  public getEpe(): NauticalMiles {
```

Comment:
These types that are just ` = number` are legacy.
```suggestion
  /**
   * Gets the estimated position error.
   * @returns Estimated position error in nautical miles, or Infinity if no position.
   */
  public getEpe(): number {
```